clientconn.go

```diff
@@ -613,12 +613,18 @@ type ClientConn struct {
 
 	// mu protects the following fields.
 	// TODO: split mu so the same mutex isn't used for everything.
-	mu              sync.RWMutex
-	resolverWrapper *ccResolverWrapper         // Always recreated whenever entering idle to simplify Close.
-	balancerWrapper *ccBalancerWrapper         // Always recreated whenever entering idle to simplify Close.
-	sc              *ServiceConfig             // Latest service config received from the resolver.
-	conns           map[*addrConn]struct{}     // Set to nil on close.
-	mkp             keepalive.ClientParameters // May be updated upon receipt of a GoAway.
+	mu sync.RWMutex
+	// nameResolutionStartTime track the start time since name resolution started.
+	nameResolutionStartTime time.Time
+	// nameResolutionInProgress indicate if name resolution is in progress.
+	nameResolutionInProgress bool
```

Comment:
I don't think we want these to be fields on the `ClientConn`.  These are supposed to be part of the RPC's lifecycle since that's what we're tracing.